rust/src/transformations/polars/make_select_trans/mod.rs

```diff
@@ -0,0 +1,273 @@
+use polars::prelude::*;
+
+use crate::core::{Function, StabilityMap, Transformation};
+use crate::domains::{Context, ExprDomain, LazyFrameDomain};
+use crate::error::*;
+use crate::metrics::{IntDistance, SymmetricDistance};
+
+/// Make a Transformation that applies list of transformations to a Lazy Frame.
+///
+pub fn make_select_trans(
+    input_domain: LazyFrameDomain,
+    input_metric: SymmetricDistance,
+    transformations: Vec<
+        Transformation<ExprDomain, ExprDomain, SymmetricDistance, SymmetricDistance>,
+    >,
+) -> Fallible<Transformation<LazyFrameDomain, LazyFrameDomain, SymmetricDistance, SymmetricDistance>> where
```

Comment:
At some point it would be nice to make this whole function generic over `MI: Metric, MO: Metric` (generics for the input metric and output metric).